scss/navbar.scss

```diff
@@ -0,0 +1,27 @@
+@import 'colors';
+$nav-height: 90px;
+$nav-logo-height: 30px;
+$text-big: 24px;
+
+.pnav-bar {
+	z-index: 10;
```

Comment:
this is not correctly indented, also z-index should not be a random number, use increasing values.